src/dotnet-core-uninstall/Windows/UninstallCommandExec.cs

```diff
@@ -48,6 +56,38 @@ public static void ExecuteUninstallCommand(IEnumerable<Bundle> bundles)
                     }
                 };
 
+                var cancelProcessHandler = new ConsoleCancelEventHandler((sender, cancelArgs) =>
+                {
+                    cancelMutex.WaitOne();
+
+                    if (!canceled)
+                    {
+                        canceled = true;
+                        Console.WriteLine(LocalizableStrings.CancelingMessage);
+                    }
+
+                    cancelArgs.Cancel = true;
+
+                    cancelMutex.ReleaseMutex();
+                });
+
+                Console.CancelKeyPress += cancelProcessHandler;
```

Comment:
This is adding the event handler each iteration of the loop.  By the Nth iteration of the loop I would expect Ctrl-C to print out N cancel messages.

We should add `Console.CancelKeyPress -= cancelProcessHandler;` following `Process.WaitForExit`.

Reply:
I agree. Will fix.